src/influxdb_ioxd/rpc/management.rs

```diff
@@ -17,6 +20,33 @@ struct ManagementService<M: ConnectionManager> {
 use super::error::{default_db_error_handler, default_server_error_handler};
 use std::num::NonZeroU32;
 
+#[derive(Debug)]
+enum UpdateError {
+    Update(server::Error),
+    Closure(tonic::Status),
+}
+
+impl From<UpdateError> for Status {
+    fn from(error: UpdateError) -> Self {
+        match error {
+            UpdateError::Update(error) => {
+                error!(?error, "Update error");
```

Comment:
I wonder if this should be 
```suggestion
                info!(?error, "Update error");
```

As I think the goal was to reserve `error!` for bugs that should be fixed in the software

Reply:
good call; changed